llvm/test/Verifier/target-ext-vector.ll

```diff
@@ -0,0 +1,20 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 5
+; RUN: opt -passes=verify -S %s | FileCheck %s
```

Comment:
Should run just llvm-as and be in test/Assembler. test/Verifier is for cases that fail